mbuild/compound.py

```diff
@@ -80,6 +83,25 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
              "distances and standard residue templates!")
         structure = pmd.load_file(filename, structure=True, **kwargs)
         compound.from_parmed(structure, coords_only=coords_only)
+
+    elif smarts:
+        pybel = import_('pybel')
+        # First we try trating filename as a SMARTS string
+        try:
+            mymol = pybel.readstring("smi", filename)
```

Comment:
Just to check, does OpenBabel expect a SMARTS or SMILES string? SMILES would make more sense to me, since the string should be describing a molecular rather than a pattern. If that's the case then we would want to change the argument in `load` from `smarts` to `smiles`.

Reply:
The babel documentation had both SMARTS and SMILES all over the place, but the distinction between pattern and molecule makes a lot of sense! I'll fix that